dipy/tracking/localtrack.pyx

```diff
@@ -86,9 +86,9 @@ cdef void fixed_step(double * point, double * direction, double step_size) nogil
 def local_tracker(
         DirectionGetter dg,
         StoppingCriterion sc,
-        cnp.float_t[:] seed_pos,
-        cnp.float_t[:] first_step,
-        cnp.float_t[:] voxel_size,
+        double[::1] seed_pos,
+        double[::1] first_step,
+        double[::1] voxel_size,
         cnp.float_t[:, :] streamline,
         double step_size,
         int fixedstep):
```

Comment:
`fixedstep` is a bool in the documentation. Can you update the type here or the documentation?

Reply:
updated. thx.